packages/devtools_app/lib/src/flutter/connect_screen.dart

```diff
@@ -39,6 +39,7 @@ class _ConnectScreenBodyState extends State<ConnectScreenBody> {
   void initState() {
     super.initState();
     controller = TextEditingController();
+    controller.text = 'http://127.0.0.1:60432/fdii_NTqdz0=/'; // XXX hack
```

Comment:
This looks like a local hack.  Please revert.

Reply:
done.Any time there is a line with XXX it indicates code that should not be checked in.